src/main.zig

```diff
@@ -1703,6 +1703,8 @@ fn buildOutputType(
                     .undefined => {
                         if (mem.eql(u8, "dynamic_lookup", it.only_arg)) {
                             linker_allow_shlib_undefined = true;
+                        } else if (mem.eql(u8, "error", it.only_arg)) {
```

Comment:
I think there is one more spot that needs updating https://github.com/ziglang/zig/blob/master/src/main.zig#L2067

Reply:
Ah yes, thanks for catching that.